CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master
 
+## 0.7.2 (2018-10-01)
```

Comment:
Please, do not update the version/date, and add new changelog entries under `master` section